docs/jwst/introduction.rst

```diff
@@ -461,6 +461,25 @@ see :ref:`Execute via call()<call_examples>`::
 For more details on the different ways to run a pipeline step, see
 the :ref:`Configuring a Step<configuring-a-step>` page.
 
+CRDS Environmentals
+-------------------
+
+The CRDS environmental variables need to be defined *before* importing anything
```

Comment:
I always used `environment` as opposed to `environmental` variable. Are they interchangeable?